baggage/baggage.go

```diff
@@ -139,6 +156,11 @@ func (p Property) Value() (string, bool) {
 // String encodes Property into a header string compliant with the W3C Baggage
 // specification.
 func (p Property) String() string {
+	//  W3C Baggage specification does not allow percent-encoded keys.
+	if !validateKey(p.key) {
+		return ""
```

Comment:
The user will NOT know about the issues occured during encoding? Do we have plan to e.g. provide new APIs for baggage propagation?

Reply:
> The user will NOT know about the issues occured during encoding?

Yes, and this is to keep the current behavior. We can add a comment to state the behavior while handing the utf8 key.

> Do we have plan to e.g. provide new APIs for baggage propagation?

Maybe provide a method called `StringRaw`? Just like what we have for `NewMember`, a method called `NewMemberRaw`.

Reply:
>  We can add a comment to state the behavior while handing the utf8 key.

It won't hurt 😉 

> Maybe provide a method called `StringRaw`? 

As far as I see, currently all implementations under https://github.com/open-telemetry/opentelemetry-go/tree/main/propagation and https://github.com/open-telemetry/opentelemetry-go-contrib/tree/main/propagators ignore errors (do not report them anywhere). So let's not expand the API further until it is clearly necessary.